knx/cemi/ldata_test.go

```diff
@@ -16,6 +16,7 @@ func makeRandBuffer(n int) []byte {
 	return buffer
 }
 
+/*
```

Comment:
Is this comment added because the function is not used? If so, please remove the function entirely.

Reply:
Commented code has been removed. I could also push the changes to `go.mod` and `go.sum` after running `go mod tidy`. The scheduled action could also be changed to twice a month, there is no real need to run it once a day. If ok I will push the modifications into this PR.